logic/src/main/nl/jft/logic/participant/impl/Team.java

```diff
@@ -1,10 +1,42 @@
 package nl.jft.logic.participant.impl;
 
+import nl.jft.common.util.Arguments;
 import nl.jft.logic.participant.Participant;
 
 /**
+ * A {@code Team} represents a group of {@link Participant Participants} that form a team.
+ * Implements the {@code Participant} interface as a {@code Team} can partake in {@link nl.jft.logic.match.Match Matches} and {@link nl.jft.logic.tournament.Tournament Tournaments}.
+ * A {@code Team} is stored in the database and is thus //TODO add identifiable link.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1135)